eos-core-depends

```diff
@@ -122,6 +122,7 @@ gnome-keyring-pkcs11
 gnome-logs
 gnome-online-accounts
 gnome-orca
+gnome-remote-desktop
```

Comment:
Hi, @jprvita! I recently claimed that GNOME screensharing works in Endless.  I'm sure it once did because I've used it, but evidently it has stopped at some point! I think you said some dependencies were deliberately excluded. This package will bring in `libvncserver1` (all its other dependencies are already installed). OK?

Reply:
Fwiw, this adds ~700kb to the ostree:
```
After this operation, 681 kB of additional disk space will be used.
```